extraConfigDpu.py

```diff
@@ -198,56 +195,39 @@ def build_dpu_operator_images() -> str:
     registry = _ensure_local_registry_running(lh, delete_all=True)
     reglocal.local_trust(lh)
 
-    operator_image = f"{registry}/{OPERATOR_IMG}"
-    daemon_image = f"{registry}/{DAEMON_IMG}"
-    render_local_images_yaml(operator_image=operator_image, daemon_image=daemon_image, outfilename=f"{REPO_DIR}/config/dev/local-images-template.yaml")
-
     lh.run_or_die(f"make -C {REPO_DIR} local-buildx")
     lh.run_or_die(f"make -C {REPO_DIR} local-pushx")
 
     return registry
 
 
-def start_dpu_operator(h: host.Host, client: K8sClient, operator_image: str, daemon_image: str, repo_wipe: bool = False) -> None:
+def start_dpu_operator(h: host.Host, client: K8sClient, repo_wipe: bool = False) -> None:
     logger.info(f"Deploying dpu operator containers on {h.hostname()}")
     if repo_wipe:
         h.run(f"rm -rf {REPO_DIR}")
         h.run_or_die(f"git clone {DPU_OPERATOR_REPO}")
-        render_local_images_yaml(operator_image=operator_image, daemon_image=daemon_image, outfilename="/tmp/dpu-local-images-template.yaml", pull_policy="IfNotPresent")
-        h.copy_to("/tmp/dpu-local-images-template.yaml", f"{REPO_DIR}/config/dev/local-images-template.yaml")
 
+    REGISTRY = host.LocalHost().hostname()
     h.run("dnf install -y pip")
     h.run_or_die("pip install yq")
     ensure_go_installed(h)
-    reglocal.local_trust(h)
-    h.run_or_die(f"podman pull {operator_image}")
-    h.run_or_die(f"podman pull {daemon_image}")
     if h.is_localhost():
         env = os.environ.copy()
         env["KUBECONFIG"] = client._kc
-        env["REGISTRY"] = host.LocalHost().hostname()
+        env["REGISTRY"] = REGISTRY
```

Comment:
By default, it will use the local host as a registry. Do we still need this?

Reply:
We do, because in the case this is running on the dpu, we want to use the prov host hostname, not the ACC hostname

Reply:
Oh wait, now I see. No we do not need this here, will remove